### PR TITLE
Update sample-type.ttl

### PR DESCRIPTION
**Pull request by KellyVance**
Updates for laboratory and field sample types and in line with submission templates.
Coherent with https://vocabs.ardc.edu.au/viewById/185
Probably needs better definition for Pulp. Feel free to offer suggestions.

### CODE REVIEW

**Comment by geoderekh** [comment]
@SalEdwards01 pls review when you get a chance

File: vocabularies/sample-type.ttl [hunk 1]
```diff
@@ -291,12 +311,19 @@ sampty:polished-block a skos:Concept ;
...
 
 sampty:powdered-chip a skos:Concept ;
-    skos:definition "A rock chip sample that has subsequently been powdered"@en ;
+    skos:definition "A rock chip sample that has subsequently been powdered."@en ;
     skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
     skos:notation "PCHIP" ;
     skos:prefLabel "Powdered Chip"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
+sampty:pulp a skos:Concept ;
```
Comment (SalEdwards01) [inline vocabularies/sample-type.ttl:320]: I have never heard of the term pulp in geology. Usually crushing, grinding and smiling are referred to for creating powdered samples.

File: vocabularies/sample-type.ttl [hunk 2]
```diff
@@ -291,12 +311,19 @@ sampty:polished-block a skos:Concept ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
 sampty:powdered-chip a skos:Concept ;
-    skos:definition "A rock chip sample that has subsequently been powdered"@en ;
+    skos:definition "A rock chip sample that has subsequently been powdered."@en ;
```
Comment (SalEdwards01) [inline vocabularies/sample-type.ttl:314]: looks good.

File: vocabularies/sample-type.ttl [hunk 3]
```diff
@@ -138,6 +144,20 @@ sampty:core a skos:Concept ;
...
+    skos:prefLabel "Half Core"@en ;
+    skos:broader sampty:core .
+
+sampty:qcore a skos:Concept ;
+    skos:definition "A core sample that has been cut in quarters parallel to its length."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
+    skos:notation "QCORE" ;
+    skos:prefLabel "Quarter Core"@en ;
+    skos:broader sampty:core .
+
```
Comment (SalEdwards01) [inline vocabularies/sample-type.ttl:160]: The letters H and P are used in combination with other letters to define core barrel diameter and therefore drillcore diameters. Will this be confusing? 

See https://www.drillingsupplystore.com/coring-in-hole-tools.html

File: vocabularies/sample-type.ttl [hunk 4]
```diff
@@ -56,6 +61,7 @@
         sampty:stream-sediment-sample,
         sampty:suspended-silt-sample,
         sampty:termite-mound,
+        sampty:thin-section,
```
Comment (SalEdwards01) [inline vocabularies/sample-type.ttl:64]: good addition

File: vocabularies/sample-type.ttl [hunk 5]
```diff
@@ -43,6 +47,7 @@
         sampty:photograph,
         sampty:polished-block,
         sampty:powdered-chip,
+        sampty:pulp,
```
Comment (SalEdwards01) [inline vocabularies/sample-type.ttl:50]: I have added a comment below where the definition in. I have not heard the term pulp used in geology.

File: vocabularies/sample-type.ttl [hunk 6]
```diff
@@ -291,12 +311,19 @@ sampty:polished-block a skos:Concept ;
...
     skos:prefLabel "Powdered Chip"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
+sampty:pulp a skos:Concept ;
+    skos:definition "A rock sample that has been pulverised sample to a very fine powder and homogenised to be representative of a parent sample, or interval. "@en ;
+    skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
+    skos:notation "PULP" ;
+    skos:prefLabel "Pulp"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
+
```
Comment (SalEdwards01) [inline vocabularies/sample-type.ttl:326]: I have not heard the term "pulp" used in geology. Usually, it is crush, grind or mill which is used to describe be the breaking down of rock into a fine grit-free powder.

File: vocabularies/sample-type.ttl [hunk 7]
```diff
@@ -388,6 +415,13 @@ sampty:termite-mound a skos:Concept ;
     skos:prefLabel "Termite Mound"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
+sampty:thin-section a skos:Concept ;
+    skos:definition "A laboratory preparation of a rock, mineral, soil, pottery, bones, or even metal sample for use with a polarizing petrographic microscope, electron microscope and electron microprobe. A thin sliver of rock is cut from the sample with a diamond saw and ground optically flat. It is then mounted on a glass slide and then ground smooth using progressively finer abrasive grit until the sample is only 30 μm thick. "@en ;
+    skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
+    skos:notation "THINS" ;
+    skos:prefLabel "Thin Section"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
```
Comment (SalEdwards01) [inline vocabularies/sample-type.ttl:423]: good addition, definition is way beyond our useage.
tech good.

File: vocabularies/sample-type.ttl (hunk 1 above)
Comment (KellyVance) [reply vocabularies/sample-type.ttl:320]: It's a mineral geochemistry thing. I'm still unsure how Pulps are different to powered rock samples, but the Minerals people assure me there is a difference.

File: vocabularies/sample-type.ttl (hunk 3 above)
Comment (KellyVance) [reply vocabularies/sample-type.ttl:160]: It shouldn't be an issue.  These notations are just for defining our URIs for each sample. The barrel diameter will be recorded separately.

File: vocabularies/sample-type.ttl (hunk 6 above)
Comment (geoderekh) [reply vocabularies/sample-type.ttl:326]: pulp is just a powdered sample and the term is mostly used in exploration and ore deposit geochemistry and assaying

File: vocabularies/sample-type.ttl (hunk 5 above)
Comment (geoderekh) [reply vocabularies/sample-type.ttl:50]: commented below

File: vocabularies/sample-type.ttl (hunk 6 above)
Comment (SalEdwards01) [reply vocabularies/sample-type.ttl:326]: If you know of it, then I'm happy :-)

File: vocabularies/sample-type.ttl (hunk 1 above)
Comment (SalEdwards01) [reply vocabularies/sample-type.ttl:320]: All good. Derek has commented that's its a term he is familiar with, so I'm good with it.

tech good.

File: vocabularies/sample-type.ttl (hunk 3 above)
Comment (SalEdwards01) [reply vocabularies/sample-type.ttl:160]: All good. Tech good.

**Pull request merged**